source/sync/data-model/update-data-model/update-data-model-production.txt

```diff
@@ -0,0 +1,431 @@
+.. _update-data-model-production:
+
+=================================
+Update a Data Model in Production
+=================================
+
+.. meta::
+   :description: Update your data model for an app in production.
+   :keywords: Realm, object model, C++ SDK, Flutter SDK, Java SDK, Kotlin SDK, .NET SDK, Node.js SDK, React Native SDK, Swift SDK
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+After your app goes to production, you might need to make additional updates to
+you data model. Production-level apps can take advantage of schema versioning,
+which supports backward-compatibility with your existing schemas
+when you make certain changes. However, updates should always be made with
+caution, as some data model changes can cause data loss and break connectivity
+with existing client devices.
+
+This feature is enabled by default for new apps created after [DATE TBD].
+
+.. note:: Enabling Schema Versioning in Older Apps
+
+   If your app backend was created *before* [DATE TBD], you must terminate,
+   then re-enable Sync to enable the schema versioning feature. To learn how,
+   refer to :ref:`pause-or-terminate-sync`.
+
+If your app is still in development, we recommend using Development Mode
+to update your data model. To learn how, refer to
+:ref:`update-data-model-development-mode`.
+
+.. _schema-versioning:
+
+Schema Versioning
+-----------------
+
+**Schema versioning** is a Device Sync feature for
+production apps that supports backward-compatibility through versioned schemas.
+This lets clients on older schemas continue to sync with the Device Sync server
+and other SDK devices, ensuring data consistency across versions.
+
+App Services handles updates to versioned schemas according to the type of change you make:
+
+- Non-breaking changes do not create a new version. They're added to your
+  current version and don't require additional handling.
+- Schema versioning changes create a new backward-compatible schema version. App
+  Services provides a server-initiated migration path for existing clients.
+- Breaking changes set a new minimum schema version. They break compatibility
+  with all previous versions and require you to manually handle reconnecting
+  clients at the new minimum version.
+
+.. important:: Schema Versioning is Incompatible with Development Mode
+
+   You must :ref:`disable Development Mode <enable-development-mode>` to use
+   schema versioning in your app. While Development Mode is enabled, App Services disables schema versioning considers schema versioning changes to be breaking changes.
+
+   If you enable Development Mode in an app with versioned schemas, the server
+   deletes all existing versions. This will break any clients using a versioned
+   schema.
+
+Non-Breaking Changes
```

Comment:
I am curious about the terminology being Non-Breaking instead of "Additive"

Reply:
Docs historically aligned on "breaking" and "non-breaking", rather than "destructive" and "additive". and it's used throughout the App Services and SDK docs